### ISSUE TITLE
No longer appearing in Chrome webstore

### ISSUE THREAD

**Issue by Quandru** [svenjacobs/sabdrop#24]
The Git appears to have recent updates but the extension no longer shows in the Chrome webstore. As a result, when rebuilding my PC, it did not install automatically as it has done previously.

I have manually exported 0.7.6 from a backup of my prevous installation and manually installed it in developer mode in Chrome but this presents a popup every time the browser loads, and the old application whitelist in the ADMX files is no longer honoured and so does not present the popup.

**Comment by svenjacobs** [comment]
Hi @AMurtagh, what do you mean by recent? The last commit on Git is from 23 Aug 2013 ;)

Unfortunately the project is no longer maintained. The source code requires adjustments to the latest changes of SABnzbd API and Chrome Web Store requirements. The extension has been removed from the Chrome Web Store by Google because it has not been adjusted to said requirements.

I updated the README to reflect the status of the project. If you want to maintain it please let me know :)

**Comment by AMurtagh** [comment]
Hi Sven,

Ah, that's a shame. It was still the best plugin I've found in terms of
reliability of working on the indexing sites I use, and still worked fine
for me.

It was still automatically installing for me and present in the web store
as recently as a month or two ago when I rebuilt my work laptop but was
gone this week/last week when I rebuilt my home PC, so hadn't had any
awareness of the project no longer being maintained!

I'm not a developer myself so no wish/ability to pick up the mantle but
I'll have a look for alternatives that work for me. Thank you for your
efforts in the past, it was a great little app!

Kind regards,


Andy Murtagh
